### PR TITLE
Separate the accessible outputs into multiple modules

### PR DESCRIPTION
**Pull request by MathuraMG**
This PR has the following changes - 
* textOutput is now accessibleOutput
* Preferences have textOutput, gridOutput and soundOutput which can be set separately
* The shortcuts Shift+Cmd+1/2 turn on and off all the accessible outputs ( + @cleezyITP). This is a temporary solution only.

Once this PR is approved, I will merge the submodule changes as well.
[Submodule PR](https://github.com/MathuraMG/p5-interceptor/pull/5)

@catarak - is there a staging/testing place this can be pushed initially?

### CODE REVIEW

**Comment by catarak** [comment]
There's not a staging server right now but I could set one up. In order to make that work, maybe it would make sense to merge this into `development`. You can also set a submodule to checkout a specific branch: http://stackoverflow.com/a/18799234/4086967. I'll get you some comments on this soon 😎

**Comment by MathuraMG** [comment]
Great, I'll make a PR to the `development` branch then first!

**Comment by catarak** [comment]
you can actually just change this pr to merge to development, by clicking "Edit" at the top of this page.

File: client/modules/IDE/components/AccessibleOutput.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (this.props.isPlaying && this.props.previewIsRefreshing) {
+      this.AccessibleOutputModal.focus();
+    }
+  }
+  render() {
+    return (
+      <section
+        className="accessible-output"
+        id="canvas-sub"
+        ref={(element) => { this.AccessibleOutputModal = element; }}
```
Comment (catarak) [inline client/modules/IDE/components/AccessibleOutput.jsx]: very minimal fix--could `this.AccessibleOutputModal` be `this.accessibleOutputModal`? I like to reserve uppercase variables for classes.

**Comment by catarak** [comment]
one small comment but otherwise this looks good to me!

**Comment by MathuraMG** [comment]
@catarak - is it okay to push this to `development` branch? I will parallely try to get write access :)

**Comment by catarak** [comment]
just added you as a collaborator--so you should be able to merge this in or push it to a separate branch if that's easier

**Comment by MathuraMG** [comment]
Woohoo, did it! Thank you @catarak

**Pull request merged**